Student/Student/Submissions/SubmissionRubric/RubricViewController.swift

```diff
@@ -56,6 +56,8 @@ class RubricViewController: UIViewController {
 
         emptyImageView.image = UIImage(named: Panda.NoRubric.name, in: .core, compatibleWith: nil)
         presenter.viewIsReady()
+        // TODO: route? 
```

Comment:
I think we could revert this file. There's a ticket to add reporting to screens where it's not available so if we have any todos we should register them there.